customers/schema.py

```diff
@@ -1,35 +1,63 @@
 import graphene
+from django.core.exceptions import ValidationError
+from django.db import transaction
 from django.utils.translation import ugettext_lazy as _
 from graphene import relay
 from graphene_django import DjangoConnectionField
 from graphene_django.filter import DjangoFilterConnectionField
 from graphene_django.types import DjangoObjectType
 from graphene_federation import extend, external
+from graphene_file_upload.scalars import Upload
 from graphql_jwt.decorators import login_required
 
 from applications.models import BerthApplication
 from applications.new_schema import BerthApplicationNode
 from berth_reservations.exceptions import VenepaikkaGraphQLError
 from leases.models import BerthLease
 from leases.schema import BerthLeaseNode
-from users.decorators import view_permission_required
+from resources.schema import update_object
+from users.decorators import (
+    add_permission_required,
+    change_permission_required,
+    delete_permission_required,
+    view_permission_required,
+)
 from users.utils import user_has_view_permission
 from utils.relay import get_node_from_global_id
 
-from .enums import InvoicingType, OrganizationType
-from .models import Boat, CustomerProfile, Organization
+from .enums import BoatCertificateType, InvoicingType, OrganizationType
+from .models import Boat, BoatCertificate, CustomerProfile, Organization
 
 InvoicingTypeEnum = graphene.Enum.from_enum(InvoicingType)
 OrganizationTypeEnum = graphene.Enum.from_enum(OrganizationType)
+BoatCertificateTypeEnum = graphene.Enum.from_enum(BoatCertificateType)
+
+
+class BoatCertificateNode(DjangoObjectType):
+    certificate_type = BoatCertificateTypeEnum(required=True)
+
+    class Meta:
+        model = BoatCertificate
+        interfaces = (relay.Node,)
+
+    def resolve_file(self, info, **kwargs):
+        if self.file:
+            return info.context.build_absolute_uri(self.file.url)
```

Comment:
nitpick, but maybe oneline it:
```python
return info.context.build_absolute_uri(self.file.url) if self.file else None
```

Reply:
I was following the example from [resources/schema.py](https://github.com/City-of-Helsinki/berth-reservations/blob/develop/resources/schema.py#L299)
```python
def resolve_image_file(self, info, **kwargs):
    if self.image_file:
        return info.context.build_absolute_uri(self.image_file.url)
    else:
        return None
```

but definitely nicer as a one-liner. 🔥

Reply:
we could also change it to a oneliner there 😬

Reply:
hm, or was it so that black formats it to several lines anyway 🤔

Reply:
Works just fine as one-liner. 👌